ts/parts/Utilities.ts

```diff
@@ -2362,11 +2368,14 @@ const setAnimation = H.setAnimation = function setAnimation(
  *         An object with at least a duration property.
  */
 const animObject = H.animObject = function animObject(
-    animation?: (boolean|Highcharts.AnimationOptionsObject)
-): Highcharts.AnimationOptionsObject {
+    animation?: (boolean|Partial<Highcharts.AnimationOptionsObject>)
+): Partial<Highcharts.AnimationOptionsObject> {
```

Comment:
This function should not return a Partial. By returning the full set, we know that `duration` is always a number.